shells/single-shell/index.html

```diff
@@ -32,6 +35,7 @@
 
   const env = new Env('../../');
   Object.assign(env.pathMap, {
+    'https://$shell/': `../../shells/`,
```

Comment:
$shell is singular but the mapping is plural, and this differs from both existing cases below. Just making sure this is intentional.

Reply:
Yes, this is a backward compatibility thing. Ty.